src/styles/blocks/card.scss

```
@@ -0,0 +1,142 @@
+@mixin column-count($count) {
+  grid-template-columns: repeat($count, 200px);
+}
+
+.main {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  @include column-count(1);
+
+  @media (min-width: 488px) {
+    & {
+      @include column-count(2);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      @include column-count(3);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      @include column-count(4);
+    }
+  }
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $gray-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: transform 0.3s ease-in-out;
+
+  &__image {
+    display: block;
+
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      text-align: left;
+      color: $secondary-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      text-align: right;
+    }
+  }
+
+  &__button {
+    content: '';
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $aqua-color;
+
+    background-color: $aqua-color;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: $white-color;
+
+    &:hover {
+      color: $aqua-color;
+      background-color: $white-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__name {
+    color: $bluish-color;
+  }
+
+  &:hover .card__button {
+    background-color: $white-color;
+    color: $aqua-color;
+  }
```

you are duplicating this data(hover for button)
```suggestion
    &:hover {
      color: $aqua-color;
      background-color: $white-color;
    }
  }

  &:hover {
    transform: scale(1.2);
  }

  &:hover .card__name {
    color: $bluish-color;
  }
```